chia/consensus/blockchain.py

```diff
@@ -6,7 +6,12 @@
 from concurrent.futures.process import ProcessPoolExecutor
 from enum import Enum
 from pathlib import Path
-from typing import Dict, List, Optional, Set, Tuple, Union
+from typing import Dict, List, Optional, Set, Tuple
+
+try:
+    from typing import Protocol
+except ImportError:
+    from typing_extensions import Protocol  # type: ignore
```

Comment:
is this according to best practice? mypy still doesn't like the type from `typing_extensions`

Reply:
Hi